cognite/client/_api/time_series.py

```diff
@@ -499,7 +499,7 @@ def create(self, time_series: TimeSeries | Sequence[TimeSeries]) -> TimeSeries |
                 >>> from cognite.client import CogniteClient
                 >>> from cognite.client.data_classes import TimeSeries
                 >>> c = CogniteClient()
-                >>> ts = c.time_series.create(TimeSeries(name="my ts"))
+                >>> ts = c.time_series.create(TimeSeries(name="my_ts", data_set_id=123, external_id="3"))
```

Comment:
Let's use strings in examples for external IDs:
```suggestion
                >>> ts = c.time_series.create(TimeSeries(name="my_ts", data_set_id=123, external_id="foo"))
```